tfjs-backend-webgpu/src/backend_webgpu.ts

```diff
@@ -120,6 +123,7 @@ export class WebGPUBackend extends KernelBackend {
         count: 2,
       });
     }
+    this.createLayouts();
```

Comment:
Maybe here we can just call
`[this.fromPixelBindGroupLayout, this.fromPixelPipelineLayout] =
        this.createTextureLayout();`
And `createLayouts()` can be removed. We can do it lazily using `getCachedOrCreateLayout` in runWebGPUProgram.